cli/packages/prisma-cli-core/src/commands/import/index.ts

```diff
@@ -33,8 +34,17 @@ export default class Import extends Command {
     if (
       this.definition.definition!.databaseType &&
       this.definition.definition!.databaseType === 'document'
-    ) {
-      throw new Error('Import is not yet supported for document stores.')
+      ) {
+        throw new Error(`Import is not yet supported for document stores. Please use the native export features of your database. 
```

Comment:
Looks good, but it says: "Please use the native **export** features" when it's for the `import` command.